src/store.ts

```diff
@@ -167,6 +168,12 @@ export function useWorkflow (): UseWorkflow {
       setGlobal(parameters)
       setFiles(newUsedFiles)
     },
+    clearNodes () {
```

Comment:
Name of function does not cover the functionality, can it be renamed to `clear()` or `clearWorkflow()`.